linera-storage/src/db_storage.rs

```diff
@@ -277,17 +277,21 @@ enum BaseKey {
     Event(EventId),
 }
 
-const INDEX_BLOB: u8 = 3;
-const BLOB_LENGTH: usize = std::mem::size_of::<BlobId>();
+const INDEX_CHAIN_ID: u8 = 0;
+const INDEX_BLOB_ID: u8 = 3;
+const CHAIN_ID_LENGTH: usize = std::mem::size_of::<ChainId>();
+const BLOB_ID_LENGTH: usize = std::mem::size_of::<BlobId>();
```

Comment:
It's actually about the BCS-serialized size, isn't it? That that's the in-memory size is just accidental.

Or rather, if we want the key to be exactly the chain/blob ID, why do we use BCS at all? Why not use the IDs' bytes directly (which is probably the same anyway)?